### PR TITLE
Update variable CAPI_VERSION to CAPM3_VERSION

### PR DESCRIPTION
**Pull request by maelk**
There is a change in metal3-dev-env that needs to be reflected here : https://github.com/metal3-io/metal3-dev-env/pull/400

### CODE REVIEW

File: _posts/2020-02-18-metal3-dev-env-install-deep-dive.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Also, `common.sh` library also sets more global variables apart from the those i
 | IRONIC_IMAGE             | quay.io/metal3-io/ironic                                  |
 | IRONIC_INSPECTOR_IMAGE   | quay.io/metal3-io/ironic-inspector                        |
 | BAREMETAL_OPERATOR_IMAGE | quay.io/metal3-io/baremetal-operator                      |
-| CAPI_VERSION             | v1alpha1                                                  |
+| CAPM3VERSION             | v1alpha3                                                  |
```
Comment (dhellmann) [inline _posts/2020-02-18-metal3-dev-env-install-deep-dive.md:121]: Should this be `CAPM3_VERSION` like it is below?

```suggestion
| CAPM3_VERSION             | v1alpha3                                                  |
```

Comment (maelk) [reply _posts/2020-02-18-metal3-dev-env-install-deep-dive.md:121]: yes indeed, sorry for the typo

**Pull request merged**